datastore/launcher-6f5c3324-4934-4d17-a68e-d2e0cd71e119/workflow_params.json

```diff
@@ -0,0 +1 @@
+{"auth_type":"none","server_base_url":"http://host.docker.internal:8089/ga4gh/drs/v1","platform_name":"ga4gh starter kit drs","report_path":"test-drs-compliance-report.json","platform_description":"GA4GH reference implementation of DRS specification"}
```

Comment:
I think this file was committed by mistake. I would remove this file to avoid any confusion.